doc/releases/quincy.rst

```diff
@@ -0,0 +1,3596 @@
+  was always installed along with the `ceph-mgr` debian package as an
+  indirect dependency. If your workflow depends on this behavior, you
+  might want to install `ceph-mgr-rook` separately.
+
+* The ``device_health_metrics`` pool has been renamed ``.mgr``. It is now
+  used as a common store for all ``ceph-mgr`` modules.
+
+* The ``ceph pg dump`` command now prints three additional columns:
+  `LAST_SCRUB_DURATION` shows the duration (in seconds) of the last completed
+  scrub;
+  `SCRUB_SCHEDULING` conveys whether a PG is scheduled to be scrubbed at a
+  specified time, whether it is queued for scrubbing, or whether it is being
+  scrubbed;
+  `OBJECTS_SCRUBBED` shows the number of objects scrubbed in a PG after a
+  scrub begins.
+
+* A health warning is now reported if the ``require-osd-release`` flag
+  is not set to the appropriate release after a cluster upgrade.
+
+* LevelDB support has been removed. ``WITH_LEVELDB`` is no longer a supported
+  build option.
+
+* Cephadm: ``osd_memory_target_autotune`` is enabled by default, which sets
+  ``mgr/cephadm/autotune_memory_target_ratio`` to ``0.7`` of total RAM. This
+  is unsuitable for hyperconverged infrastructures. For hyperconverged Ceph,
+  please refer to the documentation or set
+  ``mgr/cephadm/autotune_memory_target_ratio`` to ``0.2``.
+
+* telemetry: Improved the opt-in flow so that users can keep sharing the same
+  data, even when new data collections are available. A new 'perf' channel that
+  collects various performance metrics is now avaiable to opt into with:
+  `ceph telemetry on`
+  `ceph telemetry enable channel perf`
+  See a sample report with `ceph telemetry preview`.
+  Note that generating a telemetry report with 'perf' channel data might
+  take a few moments in big clusters.
+  For more details, see:
+  https://docs.ceph.com/en/quincy/mgr/telemetry/
+
+* MGR: The progress module disables the pg recovery event by default since the
+  event is expensive and has interrupted other services when there are OSDs
+  being marked in/out from the the cluster. However, the user can still enable
+  this event anytime. For more detail, see:
+
+  https://docs.ceph.com/en/quincy/mgr/progress/
+
+Dashboard
+~~~~~~~~~
+
+## TODO
+
+RADOS
+~~~~~
+* the "kvs" Ceph object class is not packaged anymore. The "kvs" Ceph
+  object class offers a distributed flat b-tree key-value store that
+  is implemented on top of the librados objects omap. Because there
+  are no existing internal users of this object class, it is not
+  packaged anymore.
+
+* OSD: Ceph now uses `mclock_scheduler` for bluestore OSDs as its default
+  `osd_op_queue` to provide QoS. The 'mclock_scheduler' is not supported
+  for filestore OSDs. Therefore, the default 'osd_op_queue' is set to 'wpq'
+  for filestore OSDs and is enforced even if the user attempts to change it.
+
+* MGR: The pg_autoscaler has a new 'scale-down' profile, which provides more
```

Comment:
@neha-ojha @kamoltat we probably don't need all 3 of these notes about the autoscaler - can we condense?

Reply:
agreed, I will push a commit to squash them